jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-vproduction-template.yaml

```diff
@@ -0,0 +1,37 @@
+- job-template:
+    name: 'cloud-mkcloud{version}-job-ha-vproduction-{arch}'
+    node: cloud-trigger
+    disabled: '{obj:disabled}'
+
+    triggers:
+      - timed: 'H 22 * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud{version}
+            nodenumber=4
+            clusterconfig=data+network+services=2
+            hacloud=1
+            want_all_ssl=1
+            want_dvr=1
+            want_magnum_proposal=1
+            want_barbican_proposal=0
+            want_sahara_proposal=0
+            want_murano_proposal=0
+            want_aodh_proposal=0
+            mkcloudtarget=all_noreboot
+            networkingplugin=openvswitch
+            networkingmode=vlan
+            custom_settings='export want_ceilometer_proposal=0; export want_manila_proposal=0; export want_ldap=1 ; export cephvolumenumber=3'
```

Comment:
Nope.
`custom_settings` is not made for these kinds of usages. Its intention is to be able to **manually** trigger a run with any configuration possible just for a single test.
Variables that we use regularly like in this case, should add their needed variables to the openstack-mkcloud job.